proprietary/tokens/src/brand/ams/color.tokens.json.txt

```diff
@@ -0,0 +1,130 @@
+{
```

Comment:
This feels like it should belong in an internal wiki type of place, not the repo

Reply:
Alright, removed it.